conf.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type JSONConf struct {
+	Branches struct {
+		Historical struct {
+			Development string `json:"development"`
+			//Poduction   string `json:"production"`
+		} `json:"historical"`
+		//Support struct {
+		//Feature string `json:"feature"`
+		//Release string `json:"release"`
+		//Hotfix  string `json:"hotfix"`
+		//Bugfix  string `json:"bugfix"`
+		//} `json:"support"`
+	} `json:"branches"`
+}
+
+func ReadConfiguration() JSONConf {
+	dir, _ := os.Getwd()
```

Comment:
This assumes ff is run from the repository toplevel directory, while usually git commands work also from directories inside the repository. Perhaps in the beginning of main() you could look for .git in the current directory, then in the parent, then in the parent, until you find it or you get to the root?

Also, do not ignore errors­; I see a pattern of assigning errors to _ :)

Reply:
Very good catch! I suspected something possible dangerous about position ... thank you for the suggestion.